assets/multiqc_config.yml

```diff
@@ -1,7 +1,7 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/pairgenomealign/releases/tag/1.0.0" target="_blank">nf-core/pairgenomealign</a>
+  This report has been generated by the <a href="https://github.com/nf-core/pairgenomealign/releases/tag/1.1.0" target="_blank">nf-core/pairgenomealign</a>
```

Comment:
this should be `1.1.0dev`. using `nf-core bump-version 1.1.0dev` should fix these.

Reply:
Sorry, but since my goal is to ask for a `dev` → `master` PR for a `1.1.0` release just after this PR is accepted, why should I downgrade the versions number now, and how else can I have it at `1.1.0` in master if not by staging it through `dev`?

Reply:
ah, yeah, you are correct, sorry.